draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -122,7 +122,8 @@ For any host candidate gathered by an ICE agent as part of {{RFC8445}} section 5
 
 5. Store the mDNS hostname and its related IP address in the ICE agent for future reuse.
 
-6. Replace the IP address of the ICE candidate with its mDNS hostname, and expose the candidate as usual.
+6. Replace the IP address of the ICE candidate with its mDNS hostname, expose
```

Comment:
I didn't find a way to suggest a change in a non-diff area other than creating another commit. Also used the uppercase initial to be consistent. PTAL.

Reply:
OK, LG.